spyder/plugins/editor/widgets/editor.py

```diff
@@ -1069,6 +1069,12 @@ def set_color_scheme(self, color_scheme):
             for finfo in self.data:
                 finfo.editor.set_color_scheme(color_scheme)
 
+                # Update the omst important extra selections
+                finfo.editor.unhighlight_current_line()
+                finfo.editor.unhighlight_current_cell()
+                finfo.editor.highlight_current_line()
+                finfo.editor.highlight_current_cell()
+
```

Comment:
Could here be also worty to update the highlighted occurences?

For example when going from a dark syntax to a ligh syntax you see this:

![imagen](https://user-images.githubusercontent.com/16781833/182203644-439d8740-93c0-4174-964e-aa59d983e255.png)

Instead of this:

![imagen](https://user-images.githubusercontent.com/16781833/182203714-752378e1-1b5a-4180-884e-3146415fe18c.png)

Reply:
Good catch! I'll implement it as part of this PR.

Reply:
This should be working as expected now @dalthviz.